test/controllers/users_controller_test.rb

```diff
@@ -185,6 +185,31 @@ class UsersControllerTest < ActionDispatch::IntegrationTest
     end
   end
 
+  describe 'clear_call_list method' do
+    before do
+      patch add_patient_path(@user, @patient_1), xhr: true
+      patch add_patient_path(@user, @patient_2), xhr: true
+      patch clear_call_list_path(@user), xhr: true
+      @user.reload
+    end
+
+    it 'should respond successfully' do
+      assert_response :success
+    end
+
+    it 'should remove all patients' do
+      assert_equal @user.patients.count, 0
+    end
+
+    it 'should not remove patients' do
```

Comment:
I'd like a vocabulary change so we don't confuse ourselves six months from now: can you call this test `it 'should not destroy patients'`? Remove is a word we use at a different endpoint and I'm worried I'm going to look at this in six months and be like 'wait, what?'